index.ts

```diff
@@ -33,18 +32,21 @@ app.use(
 );
 app.use('/api', api);
 app.use('/api', upload);
-app.use(
-  '/graphql',
-  graphqlHTTP({
-    schema,
-    rootValue,
-    graphiql: { defaultQuery },
-    validationRules: [queryCountLimit(5, 5)]
-  })
-);
-app.get('/*', (req, res) => res.redirect('/api'));
 
 const PORT = process.env.PORT || 3000;
-app.listen(PORT, () =>
-  console.log(`Snapshot hub started on: http://localhost:${PORT}`)
-);
+const server = new ApolloServer({
+  schema,
+  rootValue,
+  playground: true,
+  validationRules: [queryCountLimit(5, 5)]
+});
+server.start().then(() => {
```

Comment:
Do we need this? In my code i dont do any start, it look much simpler:
https://github.com/snapshot-labs/highlight/blob/577284dde1006bcbbd0258a26b0afa4c7b6343c5/src/index.ts#L14